src/platform/android.rs

```diff
@@ -80,7 +80,10 @@ pub trait EventLoopExtAndroid {}
 impl EventLoopExtAndroid for EventLoop {}
 
 /// Additional methods on [`ActiveEventLoop`] that are specific to Android.
-pub trait ActiveEventLoopExtAndroid {}
+pub trait ActiveEventLoopExtAndroid {
+    /// Get the `AndroidApp` which was used to create an event loop.
+    fn android_app(&self) -> &AndroidApp;
```

Comment:
Also unsure if we should document that `AndroidApp` is a cheaply-copyable reference type - that's already in the `android-activity` documentation though.

Reply:
I mean, users should be able to go to `AndroidApp` docs and they should be aware since they've created it.